src/client/components/ProductView/ProductView.component.js

```diff
@@ -10,8 +10,15 @@ export default function ProductView({
   categoriesList,
 }) {
   const [productsToShow, setProductsToShow] = useState(products);
-  const [currentRange, setCurrentRange] = useState(
-    productsToShow.slice(0, productsPerPage),
+  const [currentRange, setCurrentRange] = useState([]);
```

Comment:
As we discussed this ProductView should only be  used with fetched products. Soy you need loading  state in order to pass ready products than you don`t have to make this change

Reply:
Thank you Julia I will fix it today